src/GeneratedMapper.Tests/Helpers/GeneratorTestHelper.cs

```diff
@@ -43,6 +81,20 @@ public static void TestGeneratedCode(string sourceText, params string[] expected
             {
                 Assert.AreEqual(expectedOutputSourceTexts[i], output.ElementAtOrDefault(i) ?? "", $"Error in file index: {i}");
             }
+
+            if (output.Length == expectedOutputSourceTexts.Length + 2)
```

Comment:
I understand that you do this, but this can become a bit difficult when we have another feature added that might generate another file. Could you fix this with adding some flag enum which configures which default text must be added, and give it a default of `GeneratorDefaults.MapExtensions | GeneratorDefaults.ProjectToExtensions`.

Reply:
I didn't use enum since params are being used and can't have a default and params without changing all the unit tests.
What I did instead was made a DefaultFileText list, and then get count difference between expected and actual outputs and push the files onto the end so the counts match up.

This way if more files are added to the list we just need to insert the new default into DefaultFileText and it should properly check all unit tests without any additional test code changes.

Reply:
Oh yeah that is right. It's ok for now, but I can see some issues down the road when we have more files generated down the road and someone wants to specifically test one of those files. But we'll cross that bridge when we get there.